tests/quantization/test_quantized_ops.py

```diff
@@ -100,7 +100,7 @@
 def quantized_op_results_are_equal(
     quantized_op_1: QuantizedOp,
     quantized_op_2: QuantizedOp,
-    q_input: Optional[numpy.ndarray] = None,
+    q_input: Optional[numpy.ndarray] = None,  # type: ignore[arg-type]
```

Comment:
why did it change

Reply:
if it's because of the xfail, could you create an issue to remove that and add a FIXME please?
maybe not @rudy-6-4 but someone from ML team

Reply:
Because I have a mypy warning with this line, only on the CI. 
Locally, make pcc, is fine

Reply:
I did make force_licenses. 
I haven't update poetry.

Do you suggest to do first petry update --no-update then make licenses ?

Reply:
- changing the CP version shouldn't change mypy here, in my understanding
- and yes, it's not for this thread, but the poetry lock has to be done with --no-update

Reply:
we don't launch `make force_licenses.` anymore, there is an action for that now
if you don't know, maybe we can wait the rest of the team

Reply:
I followed: poetry lock no update + make licenses
mypy no longer displays errors

Reply:
yes, was certainly due to an update in mypy, which is no more updated with --no-update